packages/airnode-protocol/contracts/rrp/authorizers/admin/AirnodeStablecoinPayment.sol

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../../authorizers/interfaces/IRequesterAuthorizerWithManager.sol";
+import "./interfaces/IAirnodeFeeRegistry.sol";
+import "./interfaces/IERC20Extended.sol";
+import "./interfaces/IAirnodeStablecoinPayment.sol";
+
+/// @title The contract used to lock API3 Tokens in order to gain access to Airnodes
+contract AirnodeStablecoinPayment is IAirnodeStablecoinPayment {
+    string private constant ERROR_ZERO_CHAINID = "Zero chainId";
+    string private constant ERROR_ZERO_ADDRESS = "Zero address";
+    string private constant ERROR_NOT_AIRNODE = "Not airnode";
+    string private constant ERROR_ZERO_AMOUNT = "Zero amount";
+    string private constant ERROR_INSUFFICIENT_AMOUNT = "Insufficient amount";
+    string private constant ERROR_AIRNODE_NOT_OPTED_IN = "Airnode not opted in";
+
+    /// @notice Address of AirnodeFeeRegistry
+    address public airnodeFeeRegistry;
+
+    /// @notice The maximum whitelisting duration in seconds
+    uint64 public maximumWhitelistingDuration;
+
+    /// @notice Mapping to store the default payment address for an airnode
+    mapping(address => address) public airnodePaymentAddress;
+
+    /// @notice Mapping to store the default supported stablecoins
+    mapping(uint256 => mapping(address => bool)) public defaultSupportedERC20;
+
+    /// @notice Mapping to store the supported stablecoins for an airnode
+    mapping(uint256 => mapping(address => mapping(address => bool)))
+        public airnodeSupportedERC20;
```

Comment:
I'm now confused about why we need to have chainId in these to mappings if we are only going to support payments in mainnet stablecoins 🤔 

I mean if we are always going to make payments to an address in mainnet (airnodePaymentAddress) why do we need to check by `chainId` in the `isSupportedERC20()` modifier?

Reply:
Ah yes, we went over this yesterday. Ill change this